src/status_im/ui/screens/group/events.cljs

```diff
@@ -1,6 +1,24 @@
 (ns status-im.ui.screens.group.events
-  (:require [status-im.utils.handlers :as handlers]
-            [status-im.ui.screens.group.navigation]))
+  (:require [cljs.reader :as reader]
+            [re-frame.core :as re-frame]
+            [status-im.utils.js-resources :as js-res]
+            [status-im.utils.handlers :as handlers]
+            [status-im.ui.screens.group.navigation])
+  (:require-macros [status-im.utils.slurp :refer [slurp]]))
+
+;;;; COFX
+
+(re-frame/reg-cofx
+ :get-default-contacts
```

Comment:
why did you add this cofx?

Reply:
Just followed the pattern used for contacts. Should it be different?

Reply:
i mean `:get-default-contacts` looks like it's by mistake here, we have it in contacts ns

Reply:
Hmm merge probably.